src/commands/ProjectCommands.ts

```diff
@@ -118,13 +112,20 @@ async function chooseNewProjectDir(): Promise<string> {
 /**
  * This function is responsible for creating the project according to the chosen project type.
  *
- * @param projectPath the directory path where the project will be created
  * @param projectType the type of project the user wants to create: empty, default, or unbox a truffle project
  */
-async function createProject(projectPath: string, projectType: ProjectType): Promise<void> {
+async function createProject(projectType: ProjectType) {
+  let truffleBoxName: string;
```

Comment:
Hey @acuarica, yes, sure thing.

I've built that way because i'd like to avoid repeat this line 3 times:

`const projectPath = await chooseNewProjectDir();`

But no problem, i'll change that

Thank you!

Reply:
Oh see, I didn't realize it before. Nevermind then, it's fine as it is.